src/main/java/com/battleship/backend/models/Board.java

```diff
@@ -19,6 +19,23 @@ public String getName(){
         return name;
     }
 
+    public Sectionable getSection(int row, int col){
+        return grid[row][col];
+    }
+
+    public void addShip(Ship ship, int row, int col){
+        for(int i = 0; i < ship.getWidth(); i++){
+            addSection(ship.getShipSection(i), row, col + i);
+        }
+        for(int i = 0; i < ship.getHeight(); i++){
+            addSection(ship.getShipSection(i), row + i, col);
+        }
+    }
+
+    public void addSection(Sectionable section, int row, int col){
+        grid[row][col] = section;
+    }
+
     private void setUpBoard() {
```

Comment:
We can probably rename this to `setup` or something similar versus `setUpBoard` since this is on the `Board` object already.

Reply:
kk, will change!